src/shared/components/loading-spinner/loading-spinner.module.scss

```diff
@@ -18,16 +18,16 @@
  }
  .loading div {
    position: absolute;
-   width: 36px;
-   height: 36px;
+   width: 30px;
+   height: 30px;
    border-radius: 50%;
-   transform: translate(82px,82px) scale(1);
+   transform: translate(60px,60px) scale(1);
```

Comment:
Is `scale(1)` has effect? I think it has no... Correct me if I wrong

Reply:
You right
In this case scale has no effect
But others 'scales' in this code are important